### PR TITLE
Fix `kubectl create configmap` help

### PR DESCRIPTION
**Pull request by ncdc**
Remove "generic" from the help text, as that was an inadvertent carry-over from `kubectl create
secret generic`.

### CODE REVIEW

**Comment by ncdc** [comment]
@pmorie

**Comment by mdshuai** [comment]
@ncdc The "--type=string" also need remove
```
Usage:
  oc create configmap NAME [--type=string] [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run] [options]
```

File: docs/man/man1/kubectl-create-configmap.1 [hunk 1]
```diff
@@ -189,13 +189,13 @@ symlinks, devices, pipes, etc).
 
 .nf
   # Create a new configmap named my\-config with keys for each file in folder bar
-  kubectl create configmap generic my\-config \-\-from\-file=path/to/bar
+  kubectl create configmap my\-config \-\-from\-file=path/to/bar
 
   # Create a new configmap named my\-config with specified keys instead of names on disk
-  kubectl create configmap generic my\-config \-\-from\-file=ssh\-privatekey=\~/.ssh/id\_rsa \-\-from\-file=ssh\-publickey=\~/.ssh/id\_rsa.pub
+  kubectl create configmap my\-config \-\-from\-file=ssh\-privatekey=\~/.ssh/id\_rsa \-\-from\-file=ssh\-publickey=\~/.ssh/id\_rsa.pub
```
Comment (pmorie) [inline docs/man/man1/kubectl-create-configmap.1]: This should be something suitable for configmap, not an ssh key

File: docs/user-guide/kubectl/kubectl_create_configmap.md [hunk 2]
```diff
@@ -59,13 +59,13 @@ kubectl create configmap NAME [--type=string] [--from-file=[key=]source] [--from
 
 ```
   # Create a new configmap named my-config with keys for each file in folder bar
-  kubectl create configmap generic my-config --from-file=path/to/bar
+  kubectl create configmap my-config --from-file=path/to/bar
```
Comment (pmorie) [inline docs/user-guide/kubectl/kubectl_create_configmap.md]: I think this may also need to be updated in kubernetes.github.io

**Comment by bgrant0607** [comment]
cc @johndmulhausen re. how these docs are copied to the doc repo

**Comment by pmorie** [comment]
@ncdc needs rebase

**Comment by bgrant0607** [comment]
Ping.

**Comment by bgrant0607** [comment]
@ncdc I'd like to get this in.

cc @janetkuo

**Comment by ncdc** [comment]
@bgrant0607 I'm looking at high priority customer cases this week and don't have any time to attend to this right now. Maybe next week if that's ok?

**Comment by pmorie** [comment]
@bgrant0607 @ncdc I can take this, rebase it, and open a new PR.

**Pull request closed without merging**